### PR TITLE
Create 102-case-study-audius.md

### PR DESCRIPTION
**Pull request by jessicaschilling**
Blog post introducing the first installment in the IPFS case study series. **Intended for publication Thurs 9 July** if all approvals complete in time.

Ref for full case study text: https://github.com/ipfs/ipfs-docs/pull/366

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@autonome - If you're able to take a quick look, would really appreciate. Thanks!

File: content/post/102-case-study-audius.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+[Head over to the IPFS Docs to read the full case study](https://docs.ipfs.io/concepts/case-study-audius/), but here's a quick quote to whet your whistle:
+
+> In 2018, when Audius was looking into decentralized storage, they needed a system that would enable routing and addressing of large amounts of content in a decentralized way for their content streaming network. At the time, IPFS had the only system that fit their needs for decentralization ... Now, after two years on the network, Audius has found that IPFS has consistently performed and provided the flexibility and resilience needed for their network. "The lack of surprises is an advantage," says Hareesh Nagaraj, senior engineer at Audius. It’s been reliable the whole time."
```
Comment (momack2) [inline content/post/102-case-study-audius.md:22]:
```suggestion
> In 2018, when Audius was looking into decentralized storage, they needed a system that would enable routing and addressing of large amounts of content in a decentralized way for their content streaming network. At the time, IPFS had the only system that fit their needs for decentralization ... Now, after two years on the network, Audius has found that IPFS has consistently performed and provided the flexibility and resilience needed for their network. "The lack of surprises is an advantage," says Hareesh Nagaraj, senior engineer at Audius. "It’s been reliable the whole time."
```

File: content/post/102-case-study-audius.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+Have a look at the [full case study](https://docs.ipfs.io/concepts/case-study-audius/) now to learn how IPFS was the best fit for Audius' needs, where IPFS fits into the Audius tech stack, and the concrete gains IPFS has helped the Audius team achieve. We hope you'll be inspired!
```
Comment (momack2) [inline content/post/102-case-study-audius.md:24]:
```suggestion
Have a look at the [full case study](https://docs.ipfs.io/concepts/case-study-audius/) now to learn why IPFS was the best fit for Audius' needs, where IPFS fits into the Audius tech stack, and the concrete gains IPFS has helped the Audius team achieve. We hope you'll be inspired!
```

File: content/post/102-case-study-audius.md (hunk 1 above)
Comment (jessicaschilling) [reply content/post/102-case-study-audius.md:22]: Thanks for finding that!

**Pull request merged**